src/types/questionnaire/validation.ts

```diff
@@ -5,7 +5,7 @@ import { QuestionValidationError } from "@/types/questionnaire/batch";
 export interface FieldMetadata<T = unknown> {
   key: string;
   required: boolean;
-  validate?: (value: T) => boolean;
+  validate?: (value: T, fullValue?: unknown) => boolean;
```

Comment:
what is `fullValue`?

Reply:
Changed to `allValues`. Basically the value of the entire thing, not just the field. Let me know if you have a better suggestion